crates/oxc_minifier/src/ast_passes/peephole_replace_known_methods.rs

```diff
@@ -33,88 +33,92 @@ impl PeepholeReplaceKnownMethods {
         Self { changed: false }
     }
 
+    fn get_string_literal<'a>(expr: &Expression<'a>) -> Option<&'a str> {
+        match expr {
+            Expression::StringLiteral(lit) => Some(lit.value.as_str()),
+            Expression::TemplateLiteral(lit) if lit.is_no_substitution_template() => {
+                lit.quasi().map(|s| s.as_str())
+            }
+            _ => None,
+        }
+    }
```

Comment:
There's already a method in `ctx`, and you don't need to implement them manually again.